### DISCUSSION TITLE
[Bug] Bloody Tear "summon" Jungle Slime (Blood Moon summon item)

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1873] [Q&A] [answered]
[Bloody Tear](https://terraria.gamepedia.com/Bloody_Tear) "summon" Jungle Slime with that option in the config.json:
`"AnonymousBossInvasions": false`
It says in the chat and console logs:
> Username summoned the Jungle Slime!

**Comment by hakusaro** [comment, marked as answer]
We're now tracking this in here: https://github.com/Pryaxis/TShock/issues/1890.
